src/renderer/components/output.tsx

```diff
@@ -94,6 +94,14 @@ export const Output = observer(
             openerService: this.openerService(),
           },
         );
+        if (this.editor) {
+          this.editor.addCommand(
+            MonacoType.KeyMod.CtrlCmd | MonacoType.KeyCode.KEY_K,
+            () => {
+              if (this.model) this.model.setValue('Console Cleared !');
```

Comment:
I don't think I'd want a message when clearing the console. 😅

Reply:
is that mean i m doing right ? 
or i m doing something else which is not actually required ?
what if i do `if (this.model) this.model.setValue("');`

Reply:
and i did use this `this.model.setValue('Console Cleared !');` because i use `CTRL+L` or`CTRL+SHIFT+L`
it will move the cursor to the top of the terminal window and clear all the text that was previously displayed. 
However, it doesn't erase the command history; you can still scroll up to see previous commands and output.